doc/source/language/flow.rst

```diff
@@ -391,7 +391,10 @@ Note that any ``WAIT`` statement, no matter what the actual expression is, will
     The :ref:`WAIT <wait>` command only causes mainline code
     to be suspended.  Trigger code such as WHEN, ON, LOCK STEERING,
     and LOCK THROTTLE, will continue executing while your program
-    is sitting still on the WAIT command.
+    is sitting still on the WAIT command.  Furthermore, you cannot
```

Comment:
Not redundant.
The first form describes this:
```
when x < 0 then {
  print "x is negative."
}
set x to -1.
wait 5.  // does not stop the trigger from firing.  Still happens right away.
```
The second phrasing is describing this:
```
when x < 0 then {
  print "x is negative."
  wait 5. // this doesn't do anything.
}
set x to -1.
```